subunit.go

```diff
@@ -166,6 +186,29 @@ func (e *Event) makeTimestamp(c chan<- []byte) {
 	c <- timestamp.Bytes()
 }
 
+func (e *Event) makeMIME(c chan<- []byte) {
+	var mime bytes.Buffer
+	if e.MIME != "" {
+		writeUTF8(&mime, e.MIME)
+	}
+	c <- mime.Bytes()
+}
+
+func (e *Event) makeFileContent(c chan<- []byte) {
+	var fileContent bytes.Buffer
+	if e.FileName != "" {
+		writeUTF8(&fileContent, e.FileName)
+		writeNumber(&fileContent, len(e.FileBytes))
+		fileContent.Write(e.FileBytes)
+	}
+	c <- fileContent.Bytes()
+}
+
+func writeUTF8(b io.Writer, s string) {
```

Comment:
I agree with the errors. Let me get this merged and work in another PR with a solution for all the go routines.